docs/pulsar-api-overview.md

```diff
@@ -1,22 +1,39 @@
 ---
 id: pulsar-api-overview
-title: Pulsar API
-sidebar_label: "Overview"
+title: Pulsar APIs
+sidebar_label: "Pulsar APIs"
 ---
 
 ````mdx-code-block
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ````
 
-Pulsar is a messaging and streaming platform that scales across organizations of all sizes. Pulsar APIs are the core building blocks of Pulsar, which allow you to interact with Pulsar or administer Pulsar clusters. Pulsar APIs consist of the following types:
+Pulsar is a messaging and streaming platform that scales across organizations of all sizes. 
 
-- [Pulsar client APIs](client-api-overview.md)
-
-- [Pulsar admin APIs](admin-api-overview.md)
```

Comment:
Why delete the following contents? They are technically correct and it explains why "Pulsar client API" and "Pulsar admin API" are hosted inside this topic.

```
Pulsar APIs consist of the following types:

- [Pulsar client APIs](client-api-overview.md)

- [Pulsar admin APIs](admin-api-overview.md)
```

Reply:
It's a compromise. 
1. The links are too close (to the following two targets) to redirect, especially for the admin API section (newly added heading2), which provides another link to the overview topic. It's just not a good idea to let users click twice for a single piece of information. So I select to keep the link under the heading2.
2. Both the illustration and the following headings can help them get to this point.

Reply:
Thanks for your suggestions added in the screenshot. I can move the API overview to the top, but I prefer not to add a standalone heading4 for REST APIs inside the topic. Similar to the bullet style, a single one doesn't make too much sense but has the chance to confuse users. We can revisit this if we have at least two items in parallel.

Reply:
> a single one doesn't make too much sense but has the chance to confuse users.

The parallel structure (Pulsar APIs and REST APIs) confuses users as well.

If not embedding it to the "Pulsar APIs" topic, how about optimizing the left TOC as below?

```
# Reference

## Pulsar APIs
### REST APIs

## Terminology
## Pulsar CLI tools
## ...
```

Reply:
> The parallel structure (Pulsar APIs and REST APIs) confuses users as well.

Yes, I agree. I'll change the sidebar label as a simple way to reconcile.